<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventarioVer</class>
 <widget class="QDialog" name="InventarioVer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar / Editar Producto</string>
  </property>
  <widget class="QLabel" name="foto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>178</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="cancel_prod">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="update_prod">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Subir Imagen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_categoria">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_precioPublico">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Publico:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_precioFabrica">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Fabrica:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cantidad_minima">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad Minima:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cantidad">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_descripcion">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripcion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_codigoBarras">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Codigo Barras:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_producto">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="producto"/>
    </item>
    <item>
     <widget class="QLineEdit" name="codigoBarras"/>
    </item>
    <item>
     <widget class="QTextEdit" name="descripcion"/>
    </item>
    <item>
     <widget class="QLineEdit" name="cantidad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="cantidad_minima"/>
    </item>
    <item>
     <widget class="QLineEdit" name="categoria"/>
    </item>
    <item>
     <widget class="QLineEdit" name="precioPublico"/>
    </item>
    <item>
     <widget class="QLineEdit" name="precioFabrica"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>foto</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_categoria</zorder>
  <zorder>label_precioPublico</zorder>
  <zorder>label_precioFabrica</zorder>
  <zorder>label_cantidad_minima</zorder>
  <zorder>label_cantidad</zorder>
  <zorder>label_descripcion</zorder>
  <zorder>label_codigoBarras</zorder>
  <zorder>label_producto</zorder>
  <zorder>label_id</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
